### DISCUSSION TITLE
standard_init_linux.go:211: exec user process caused "exec format error"

### DISCUSSION THREAD

**Discussion by magic3584** [dani-garcia/vaultwarden#1754] [Q&A]
I'm running on a ARM device, but it show **standard_init_linux.go:211: exec user process caused "exec format error"** when using caddy.
Is it support ARM devices?

**Comment by RealOrangeOne** [comment]
Vaultwarden supports ARM, yes:

![image](https://user-images.githubusercontent.com/6527489/120978984-ff62ea00-c76c-11eb-8af8-8fe3f85547d8.png)

So too does Caddy

If you're getting that error when trying to run caddy, then best report it to caddy, not here. Also would be helpful to know what command you're running when you get that.

**Reply by magic3584** [reply]
Thanks, I'm following this: https://www.linode.com/docs/guides/how-to-self-host-the-bitwarden-rs-password-manager/
But when I run on another Debian server, I'm stuck with nginx, sad
